src/Generator/Passes/GetterSetterToPropertyPass.cs

```diff
@@ -247,7 +249,9 @@ private static string GetPropertyNameFromSetter(Method setter)
 
             private void DistributeMethod(Method method)
             {
-                if (method.OriginalReturnType.Type.IsPrimitiveType(PrimitiveType.Void))
+                if (method.OriginalReturnType.Type.IsPrimitiveType(PrimitiveType.Void) ||
+                    (method.OriginalReturnType.Type.IsPrimitiveType(PrimitiveType.Bool) &&
+                     method.Parameters.Count > 1))
```

Comment:
More than 1 parameter? How is this supposed to work with setters only having one?